### ISSUE TITLE
Missing tag models

### ISSUE THREAD

**Issue by Hussell** [MotusDev/Motus-TO-DO#513]
Last night a batch of tags failed to register on our end because the tag model wasn't in lk_tag_models. Comparing that table to the list in https://github.com/MotusWTS/motusServer/blob/master/inst/doc/modelLotekTagLifeSpan.Rmd, I see the following differences:

Missing from lk_tag_models:
ANTC-M3-1
ANTCW-M3-1
NTQB2-4-2

I'm assigning this to Denis because I can't update the table and Catherine is in the field.

**Comment by denislepage** [comment]
I have added NTQB2-4-2 based on the same specs as NTQB2-4-2S which we already had. We'll need to confirm with Lotek what the actual specs are. Jeremy has confirmed that this is the one the user was trying to register, but they ended up using NTQB2-4-2S instead as a work-around.

I didn't add the 2 ANTC tags for now. 

I'll flag Catherine as something for her to tackle when she's back.

It's be nice to streamline a bit this process with Lotek when new tags are issued.

**Comment by Hussell** [comment]
Ah, beg your pardon, but the user (Kristie Stein) substituted NTQB-4-2 instead as a work-around. "Today, I tried changing the tag model from "NTQB2-4-2" (as listed on the Lotek invoice) to "NTQB-4-2" (as it is listed on the Lotek website)."

Project 247, tags IDs 35980 through 35991 (12 tags).

**Comment by cjardine-bsc** [comment]
I've updated those 12 tags from project_id 247 to NTQB2-4-2. So those should now be correct. 

The two versions NTQB2-4-2S and NTQB2-4-2 appear to use the same battery, so SG is using the same lifespan specs for them. 

There is one mention of 3-1 from https://github.com/MotusWTS/motusServer/blob/master/inst/doc/modelLotekTagLifeSpan.Rmd 

"and there are additional models listed in "ANTC Spec Sheet.pdf" (no
source), including the only numbers for battery type "3-1". "

I found those specs and ran the model to get D ant rt 

ANTC-M3-1 | 138.51764 | 11.78533

And they've been added to the DB.
